cas-server-core-util/src/main/java/org/jasig/cas/util/ResourceUtils.java

```diff
@@ -0,0 +1,42 @@
+package org.jasig.cas.util;
+
+import org.springframework.core.io.AbstractResource;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.FileSystemResource;
+import org.springframework.core.io.UrlResource;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+
+/**
+ * Utility class to assist with resource operations.
+ *
+ * @author Misagh Moayyed
+ * @since 4.3.0
+ */
+public final class ResourceUtils {
+
+    private ResourceUtils() {}
+
+    /**
+     * Gets resource from a String location.
+     *
+     * @param metadataLocation the metadata location
+     * @return the resource from
+     * @throws IOException the exception
+     */
+    public static AbstractResource getResourceFrom(final String metadataLocation) throws IOException {
```

Comment:
There are converters that take a String and turn it into a bean, but as far as I know, they work with xml config manipulation.